app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-expand-lg navbar-dark bleu">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">Expended</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link" aria-current="page" href="<%= groups_path %>">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="<%= new_group_path %>">Create group</a>
+        </li>
+
+        <li class="nav-item">
```

Comment:
[OPTIONAL]
- The create expend button should be placed at the bottom like in the original design. When you get the time, please do it.
![image](https://github.com/francksefu/my_expended/assets/45460192/019aa3f7-934b-4350-9e8a-c7aaa219a081)